.babelrc

```diff
@@ -5,6 +5,8 @@
   ],
   "plugins": [
     "lodash",
-    "transform-class-properties"
+    "transform-class-properties",
+    "transform-es2015-destructuring",
+    "transform-object-rest-spread"
```

Comment:
I do think we should remove this one. Especially since it's not even in the es2016 spec, it's still in draft stage 3, so it's possible it'll change